test/tests_for_integration/api_to_db_test.py

```diff
@@ -173,7 +210,22 @@ def ee2_port(service, mongo_client, config):
     yield service
 
 
-def test_is_admin(ee2_port):
-    ee2cli = ee2client("http://localhost:" + ee2_port)
-    print(ee2cli.status())
-    # TODO add a test
+def test_is_admin_success(ee2_port):
+    ee2cli_read = ee2client("http://localhost:" + ee2_port, token=TOKEN_READ_ADMIN)
+    ee2cli_no = ee2client("http://localhost:" + ee2_port, token=TOKEN_NO_ADMIN)
+    ee2cli_write = ee2client("http://localhost:" + ee2_port, token=TOKEN_WRITE_ADMIN)
+
+    # note that if we ever need to have Java talk to ee2 these responses will break the SDK client
```

Comment:
Ha! Who cares about the SDK client anyway?!

Reply:
<--- this guy here